src/libraries/Common/src/System/Obsoletions.cs

```diff
@@ -42,5 +42,8 @@ internal static class Obsoletions
 
         internal const string CodeBaseMessage = "Assembly.CodeBase and Assembly.EscapedCodeBase are only included for .NET Framework compatibility. Use Assembly.Location instead.";
         internal const string CodeBaseDiagId = "SYSLIB0012";
+
+        internal const string EscapeUriStringMessage = "Uri.EscapeUriString can corrupt the Uri string in some cases. Consider using Uri.EscapeDataString instead.";
+        internal const string EscapeUriStringDiagId = "SYSLIB0013";
```

Comment:
This ID is being claimed by #41400

Reply:
I changed it in #41400 as it seems this one has bigger chance to be merged first.